performance/benchmarks/micro_benchmarks.rb

```diff
@@ -31,6 +32,43 @@ def divider
 
 divider
 
+# Time Threshold: 1 second
```

Comment:
My data from quickly running locally:

## This PR (after pagination change)
- 0.150s
- 0.161s
- 0.153s

## 1.28.0
- 3.353s
- 3.390s
- 3.300s

Reply:
thanks for running some benchmarks on it, I ran it locally too and it seemed to be in the low 0.1s for me, so I set the threshold to 0.2s for now